### PR TITLE
[EasyActivity] ActivityLogEntryMessage does not implement WithLockDataInterface

### PR DESCRIPTION
**Pull request by mvlad**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

**Review by roman-eonx — APPROVED** [review]

**Review by itorgov — APPROVED** [review]

File: packages/EasyActivity/src/Bridge/Symfony/Messenger/ActivityLogEntryMessage.php [hunk 1]
```diff
@@ -5,9 +5,16 @@
 namespace EonX\EasyActivity\Bridge\Symfony\Messenger;
 
 use EonX\EasyActivity\ActivityLogEntry;
+use EonX\EasyLock\Interfaces\LockDataInterface;
+use EonX\EasyLock\Interfaces\WithLockDataInterface;
+use EonX\EasyLock\LockData;
 
-final class ActivityLogEntryMessage
+final class ActivityLogEntryMessage implements WithLockDataInterface
```
Comment (natepage) [inline packages/EasyActivity/src/Bridge/Symfony/Messenger/ActivityLogEntryMessage.php:12]: will that fail if EasyLock isn't installed?

**Review by natepage — CHANGES_REQUESTED** [review]

File: packages/EasyActivity/src/Bridge/Symfony/Messenger/ActivityLogEntryMessage.php (hunk 1 above)
Comment (mvlad) [reply packages/EasyActivity/src/Bridge/Symfony/Messenger/ActivityLogEntryMessage.php:12]: Yes, the following error occurs without EasyLock:
```
Error: Interface "EonX\EasyLock\Interfaces\WithLockDataInterface" not found
```
https://github.com/eonx-com/easy-monorepo/runs/8109257071?check_suite_focus=true#step:7:17.

Comment (roman-eonx) [reply packages/EasyActivity/src/Bridge/Symfony/Messenger/ActivityLogEntryMessage.php:12]: @natepage we suggest installing this package in `composer.json`, similar to `symfony/messenger`. A project can use the package without async logic or with its own implementation,

Comment (natepage) [reply packages/EasyActivity/src/Bridge/Symfony/Messenger/ActivityLogEntryMessage.php:12]: @roman-eonx so this message isn't triggered by default? do we have like a config option to enable async?

Comment (roman-eonx) [reply packages/EasyActivity/src/Bridge/Symfony/Messenger/ActivityLogEntryMessage.php:12]: @natepage `\EonX\EasyActivity\Bridge\Symfony\Messenger\AsyncDispatcher` is the default implementation of `\EonX\EasyActivity\Interfaces\AsyncDispatcherInterface`. And `\EonX\EasyActivity\Logger\AsyncActivityLogger` is the default implementation of `\EonX\EasyActivity\Interfaces\ActivityLoggerInterface`. Both are set in the Symfony bridge config. So, this is the default configuration for a Symfony project. But a project can implement its own dispatcher without Messenger or without locking. Or even use synchronous dispatching.

Comment (natepage) [reply packages/EasyActivity/src/Bridge/Symfony/Messenger/ActivityLogEntryMessage.php:12]: ok thank you for the explanation

**Review by natepage — APPROVED** [review]

**Pull request merged**